op-chain-ops/upgrades/l1.go

```diff
@@ -107,7 +108,26 @@ func L1CrossDomainMessenger(batch *safe.Batch, implementations superchain.Implem
 		return err
 	}
 
-	calldata, err := l1CrossDomainMessengerABI.Pack("initialize", superchainConfigProxy)
+	var optimismPortal, otherMessenger common.Address
+	if config != nil {
+		optimismPortal = common.HexToAddress(list.OptimismPortalProxy.String())
```

Comment:
Same comment as other PRs, prefer to pull remote values and then check them for equality against the config if the config is present